pkg/kubecost/kubecost_codecs.go

```diff
@@ -1426,6 +1427,9 @@ func (target *AllocationProperties) UnmarshalBinaryWithContext(ctx *DecodingCont
 	target.Annotations = AllocationAnnotations(tt)
 	// --- [end][read][alias](AllocationAnnotations) ---
 
+	eee := buff.ReadBool() // read bool
+	target.AggregatedMetadata = eee
```

Comment:
Is aggregated metadata supposed to be part of this PR?

Reply:
fixed, this is ignored